bench/formal/Makefile

```diff
@@ -1,8 +1,12 @@
 ###########################################################
-# This Source Code Form is subject to the terms of the
-# Open Hardware Description License, v. 1.0. If a copy
-# of the OHDL was not distributed with this file, You
-# can obtain one at http://juliusbaxter.net/ohdl/ohdl.txt
+# This source describes Open Hardware and is licensed under the CERN-OHLW v2
+#
+# You may redistribute and modify this documentation and make products
+# using it under the terms of the CERN-OHL-W v2 (https:/cern.ch/cern-ohl).
+# This documentation is distributed WITHOUT ANY EXPRESS OR IMPLIED
+# WARRANTY, INCLUDING OF MERCHANTABILITY, SATISFACTORY QUALITY
+# AND FITNESS FOR A PARTICULAR PURPOSE. Please see the CERN-OHL-W v2
+# for applicable conditions.
```

Comment:
Rather than putting the full text in each heading should we just put the SPX identifier?  This is what they moved to in the linux kernel.  i.e. In the kernel we have:

```
// SPDX-License-Identifier: GPL-2.0-or-later
/*
 * OpenRISC setup.c
 *
 * Linux architectural port borrowing liberally from similar works of
 ...
```

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/openrisc/kernel/setup.c?h=v5.16-rc2#n1

Reply:
Yeah I like this. Let me rework the PR. Hopefully won't take me another 3 months to get around to it 🙈

Reply:
Hi @juliusbaxter , did you ever get around to updating the PR?

Reply:
I did it.